src/components/MyMealPlan.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+class MyMealPlan extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
```

Comment:
ditto file issue and put TODO

Reply:
so what you do is you file the issue, and then here you go:

// TODO(issue/12): Meal plan component

or something like that